CHANGES.rst

```diff
@@ -14,6 +14,8 @@ esa.euclid
 
 - New module to access the ESA Euclid Archive. [#3216]
 
+- Tests remove temp directories. [#3226]
+
```

Comment:
no need for changelog entry here for multiple reasons: 1) we don't changelog internal (user invisible), docs, or test changes 2) the PR it follows up is not yet released, so these changes are "affect-dev". If there are new functionality added we may just list the PR number under the previous entry.